### PR TITLE
manaplus: init at 1.9.3.23

### PR DESCRIPTION
**Pull request by lheckemann**
###### Things done

- [x] Ran the game
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/games/manaplus/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchurl, SDL2, SDL2_image, SDL2_ttf, SDL2_mixer, SDL2_net,
+SDL2_gfx, zlib, physfs, curl, libxml2, libpng, pkgconfig, mesa, autoreconfHook
+}:
+stdenv.mkDerivation rec {
```
Comment (aanderse) [inline pkgs/games/manaplus/default.nix:4]: Please include `meta`.

File: pkgs/games/manaplus/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  version = "1.9.3.23";
+  name = "manaplus-" + version;
+
+  src = fetchurl {
+    url = "https://download.evolonline.org/manaplus/download/${version}/manaplus-${version}.tar.xz";
+    sha256 = "1ky182p4svwdqm6cf7jbns85hidkhkhq4s17cs2p381f0klapfjz";
+  };
+
+  buildInputs = [
```
Comment (aanderse) [inline pkgs/games/manaplus/default.nix:13]: `autoreconfHook` and `pkgconfig` belong under `nativeBuildInputs`.

Also I believe `pkg-config` is preferred to `pkgconfig` now.

**Review by aanderse — CHANGES_REQUESTED** [review]

File: pkgs/games/manaplus/default.nix [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchurl, SDL2, SDL2_image, SDL2_ttf, SDL2_mixer, SDL2_net,
+SDL2_gfx, zlib, physfs, curl, libxml2, libpng, pkgconfig, mesa, autoreconfHook
+}:
+stdenv.mkDerivation rec {
+  version = "1.9.3.23";
+  name = "manaplus-" + version;
```
Comment (SuperSandro2000) [inline pkgs/games/manaplus/default.nix:6]:
```suggestion
  pname = "manaplus";
  version = "1.9.3.23";
```

**Comment by lheckemann** [comment]
Oops, just picked up this ancient package and bumped the version — forgot all about checking to see if it's up to standards :facepalm:

**Comment by aanderse** [comment]
I figured it was either that or your github account got hacked 😆 

I was mostly shocked to learn that software originally stemming from `themanaworld` is still active. And then more shocked to learn `themanaworld` itself is still active 😄

**Comment by lheckemann** [comment]
Addressed the comments, also fixed `mesa` → `libGL`

And yes, it's even grown! At least the maps have :D

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107245` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>manaplus</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107245` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>manaplus</li>
  </ul>
</details>

**Pull request merged**